cmd/genericconf/config.go

```diff
@@ -70,3 +70,36 @@ func ParseLogType(logType string) (log.Format, error) {
 	}
 	return nil, errors.New("invalid log type")
 }
+
+type FileLoggingConfig struct {
+	Enable     bool   `koanf:"enable"`
+	File       string `koanf:"file"`
+	MaxSize    int    `koanf:"max-size"`
+	MaxAge     int    `koanf:"max-age"`
+	MaxBackups int    `koanf:"max-backups"`
+	LocalTime  bool   `koanf:"local-time"`
+	Compress   bool   `koanf:"compress"`
+	BufSize    int    `koanf:"buf-size"`
+}
+
+var DefaultFileLoggingConfig = FileLoggingConfig{
+	Enable:     false,
+	File:       "logs",
```

Comment:
Could we put this adjacent to the `nitro` database folder?

Reply:
is `node.DefaultConfig.DataDir` the folder?

Reply:
We should call `stack.ResolvePath(loggingConfig.File)` to figure out the path